module/observations/helper.js

```diff
@@ -61,77 +63,117 @@ module.exports = class ObservationsHelper {
      * @name create
      * @param {String} solutionId -solution id.
      * @param {Object} data - Observation creation data.
-     * @param {Object} userDetails - Logged in user details.
-     * @param {String} requestingUserAuthToken - Requesting user auth token. 
+     * @param {Object} userId - Logged in user id.
+     * @param {String} requestingUserAuthToken - Requesting user auth token.
+     * @param {String} [programId = ""] - program id
      * @returns {Object} observation creation data.
      */
 
-    static create(solutionId, data, userDetails, requestingUserAuthToken = "") {
+    static create(
+        solutionId,
+        data, 
+        userId, 
+        requestingUserAuthToken = "",
+        programId = ""
+    ) {
         return new Promise(async (resolve, reject) => {
             try {
 
-                if(requestingUserAuthToken == "") {
+                if( requestingUserAuthToken == "" ) {
                     throw new Error(messageConstants.apiResponses.REQUIRED_USER_AUTH_TOKEN);
                 }
 
-                let userOrganisations = await shikshalokamHelper.getUserOrganisation(requestingUserAuthToken, userDetails.id);
-                let createdFor = new Array;
-                let rootOrganisations = new Array;
+                let organisationAndRootOrganisation = 
+                await shikshalokamHelper.getOrganisationsAndRootOrganisations(
```

Comment:
@aman-tunerlabs - No check whether or not the organizations are fetched from sunbird ?